.github/workflows/buildAndTest.yml

```diff
@@ -73,13 +73,14 @@ jobs:
     - name: Generate documentation
       if: steps.cache-llvm.outputs.cache-hit == 'true'
       run: |
-        mkdir llhd/build
-        cd llhd/build
-        cmake $GITHUB_WORKSPACE/llhd -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_LINKER=/usr/bin/lld -DMLIR_DIR=$GITHUB_WORKSPACE/llvm/install/lib/cmake/mlir/ -DLLVM_DIR=$GITHUB_WORKSPACE/llvm/install/lib/cmake/llvm/ -DLLVM_EXTERNAL_LIT=$GITHUB_WORKSPACE/llvm/build/bin/llvm-lit
+        mkdir $GITHUB_WORKSPACE/llhd/build
+        cd $GITHUB_WORKSPACE/llhd/build
+        cmake $GITHUB_WORKSPACE/llhd -DCMAKE_BUILD_TYPE=Release -DCMAKE_LINKER=/usr/bin/lld -DMLIR_DIR=$GITHUB_WORKSPACE/llvm/install/lib/cmake/mlir/ -DLLVM_DIR=$GITHUB_WORKSPACE/llvm/install/lib/cmake/llvm/ -DLLVM_EXTERNAL_LIT=$GITHUB_WORKSPACE/llvm/build/bin/llvm-lit
```

Comment:
Is there a reason why the test build uses clang, while the doc build doesn't?

Reply:
The documentation build should not rely on anything from the compiler. Either we add it and it is never used or we drop it and we can also remove the `CMAKE_LINKER` definition. But yes, there is something to change in the patch here.

Reply:
I see. I originally copy pasted the above step when separating the doc generation from the test build. So we could drop everything but `DMLIR_DIR` and `DLLVM_DIR`, if I understand correctly?

Reply:
Yes, it should work with only these two definitions.

Reply:
Could you make the changes on your side? I don't have access to my local copy of the repository.